scripts/build_installer.ps1

```diff
@@ -9,11 +9,12 @@ function DownloadLanguageFile($languageFile, $innoPath) {
 
 $innoInstallPath = 'C:\Program Files (x86)\Inno Setup 6'
 
-# Install the unofficial Korean, Simplified Chinese and Swedish translation
-# files for Inno Setup.
+# Install the unofficial Bulgarian, Korean, Swedish and Simplified Chinese
+# translation files for Inno Setup.
+DownloadLanguageFile '\Bulgarian.isl'  $innoInstallPath
```

Comment:
@Ortham Should adding this line also be discussed in CONTRIBUTING.md for adding unofficial translations?

Reply:
Yes, I think so.